frontend/public/texts/getHubTexts.js

```diff
@@ -120,6 +120,14 @@ export default function getHubTexts({ hubName }) {
       en: "Add Hubs (sectors) in which your organization is active",
       de: "Füge Hubs (Sektoren) hinzu, in denen deine Organisation aktiv ist",
     },
+    please_create_an_account_or_log_in_to_contact_the_ambassador: {
+      en: `Please sign up to contact ${localAmbassador?.first_name}.`,
+      de: `Bitte melde dich an, um ${localAmbassador?.first_name} zu kontaktieren.`,
+    },
+    contact_ambassador: {
+      en: "Contact Local Ambassador",
+      de: "Lokaler Klimavernetzer Kontaktieren",
```

Comment:
I'd make this generic, e.g. if the local ambassador is a woman it would be "Klimavernetzerin" in german. We could just pull this from the title
```suggestion
      en: `Contact ${localAmbassador.title}`,
      de: `${localAmbassador.title} kontaktieren`,
```